### ISSUE TITLE
A11y_SQL Azure Data Studio_Import Extension - Specify Input File_ScreenReader: The heading step1 and import flat file wizard are both announced as heading 1 in browse mode.

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#4163]
“[Check out Accessibility Insights](https://accessibilityinsights.io/)! – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** -A11yMAS;-A11yTCS;-A11ySev2;-BM_SQLAzureDataStudio_Oct2020; -SQL Azure Data Studio;-Win10;-FTP;-DesktopApp;- WCAG1.3.1;-Benchmark;-NVDA;-AILimited;

**Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10
Screen reader: NVDA2020.3

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to the connected server and press enter key, use down key and select any database keep_MDW use the right-click key on the keyboard on it and select import wizard.
4. Launch screen reader as NVDA and navigate to heading step1 and heading "import flat file wizard" in browse mode insert +7 from drop down "server the database is in"

**Actual:**
On navigating to heading "step1" and heading "import flat file wizard" in browse mode insert +7 from drop down "server the database is in" both the headings are announced as heading 1.

**Issue observed with JAWS**

**Expected:**
 Heading "import flat file wizard" should be heading 1 and heading "step1" should be heading 2 thus making heading "specify input file" heading 3

**User Impact:**
The screen reader users will get the wrong page structure.

**Attachment:**
